### DISCUSSION TITLE
Custom stylesheet

### DISCUSSION THREAD

**Discussion by bkalbfus** [de-jcup/eclipse-asciidoctor-editor#379] [Q&A]
It's great to have instant preview of an `.adoc` file!   I want to be able to prototype my document completely, including custom styles (do we call them "custom roles" here?),  and margins/page breaks.  I added the declaration at top of my file:
`:stylesheet: My_custom_stylesheet.css`
and I see that the `{docdir}`  and `{stylesdir}` would seem to point to my css file that is in the same directory as my source file.  I do not see the styles being applied.

I am using the embedded ASP.  Is this something that would need to be overridden before the document gets parsed?  If so, having the ability to do it with the embedded ASP would be helpful.

Thanks!
Brian

**Comment by de-jcup** [comment]
@bkalbfus : sorry for responding here so late - I totally missed the discussions.

To be honest, I have currently not used style sheets but instead always relied on the standard styling from asciidoctor.

After reading https://docs.asciidoctor.org/asciidoctor.js/latest/extend/stylesheets/  I think that using [Asciidoctor config files](https://github.com/de-jcup/eclipse-asciidoctor-editor/wiki/Asciidoctor-configfiles) and setup the wanted stylesheet setting there, could solve the problem.

If this does not work, please give me feedback and I will create an issue or transform this discussion to an issue.
